include/boost/geometry/algorithms/detail/overlay/get_turns.hpp

```diff
@@ -143,7 +143,7 @@ class get_turns_in_sections
 
     template <typename Geometry, typename Section>
     static inline bool neighbouring(Section const& section,
-            int index1, int index2)
+            signed_index_type index1, signed_index_type index2)
```

Comment:
I thought we should rename signed_index_type to signed_size_type and that it was already done (at least in buffer),
so I don't understand why signed_index_type is now popping up here again...